README.md

```diff
@@ -1,126 +1,128 @@
+     # Set this to true If you want to give permission at organization level
+     Set this to false otherwise (If you want to give permissions per child project)
+     set_org_level_permissions = true
 
-### 2. Init, Plan and Apply
+     # AWS account details
+     # Copy Uptycs's AWS Account ID and Role from Uptycs' UI.
+     # Uptycs' UI: "Cloud"->"GCP"->"Integrations"->"ORGANIZATION INTEGRATION"
+     host_aws_account_id     = "<AWS account id>"
+     host_aws_instance_roles  = ["Role_Allinone", "Role_PNode", "Role_Cloudquery"]
+   }
 
-#### Inputs
+   output "host-project-id" {
+     value = module.create-gcp-cred.host-project-id
+   }
 
+   output "integration-name" {
+     value = module.create-gcp-cred.integration-name
+   }
 
-| Name                      | Description                                                           | Type           | Default                 |
-| --------------------------- | ----------------------------------------------------------------------- | ---------------- | ------------------------- |
-| organization_id           | The GCP parent organizations id where resources will be created.      | `string`       | Required                |
-| integration_name          | Unique phrase. Used to name resources                                 | `string`       | `"uptycs-int-20220101"` |
-| service_account_name      | The service account name which will be created in host project.       | `string`       | `"sa-for-uptycs"`       |
-| host_project_id           | GCP Project ID that Uptycs should create required resources under     | `string`       | Required                |
-| host_aws_account_id       | AWS account id of Uptycs - for federated identity                     | `string`       | Required                |
-| host_aws_instance_roles   | AWS role names of Uptycs - for identity binding                       | `list(string)` | Required                |
-| set_org_level_permissions | The flag to choose permissions at organization level or project level | `bool`         | true                |
+   output "credential-file-gen-command" {
+     value = module.create-gcp-cred.credential-file-gen-command
+   }
 
-#### Outputs
+   ```
 
+**Notes**:
 
-| Name                        | Description                                |
-| ----------------------------- | -------------------------------------------- |
-| credential-file-gen-command | Command to generate credentials JSON file. |
-| host-project-id             | Host Project ID.                           |
+- **If you set the flag `set_org_level_permissions` to `true`, the permissions at the organization level are attached to the service account. Any addition of a project to the organization is automatically integrated with Uptycs. You do not need to execute `step 2`.**
```

Comment:
This is very confusing.
Are trying to convey that user has to re-run the config if there are new-projects for cases where they did not give us org-permissions?
If so, delete these two lines. Our UI will guide them to re-run TF.
Other option is to add a small note at the end to re-run terraform when new projects are created.
Again having this comment here is very confusing.